almalinux8-multi-upgrade.sh

```diff
@@ -10,8 +10,8 @@ echo ""
 echo "= almalinux1 ========================================================================"
 vagrant ssh almalinux1 -c "sudo /banzaicloud/pke upgrade master --kubernetes-version='$KUBERNETES_VERSION'"
 
-# waiting 10 seconds because of apiserver
-sleep 10
+# waiting 20 seconds because of apiserver
+sleep 20
```

Comment:
:(

Reply:
:( in the case of almalinux multi-master, the 10 seconds wasn't enough each time. So sad.